src/Bindings/ManualBindings.cpp

```diff
@@ -3684,6 +3684,48 @@ static int tolua_cEntity_GetSpeed(lua_State * tolua_S)
 
 
 
+static int tolua_cColor_Set(lua_State * tolua_S)
+{
+	cLuaState L(tolua_S);
+
+	if (!L.CheckParamNumber(2, 4) || !L.CheckParamSelf("cColor"))
+	{
+		return 0;
+	}
+	cColor * self = reinterpret_cast<cColor *>(tolua_tousertype(tolua_S, 1, nullptr));
```

Comment:
It's not failing because it's typed, it's failing because it's anonymous.  Just give it a name like `eColorConstants` and add `// tolua_export` to the closing curly for the class and it should work.